src/components/SecondaryNav/__tests__/SeconadryNav-tests.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import SecondaryNav, { NavLink } from '../SecondaryNav';
+
+it(`secondary navigation renders correctly`, () => {
+    const tree = renderer.create(
+      <SecondaryNav
+        buttons={[<NavLink key={1}><a href="/">Testing</a></NavLink>]}
```

Comment:
Shouldn't `NavLink` be an actual anchor or button element and not require a nested anchor tag?

Reply:
Because `NavLink` is a named export, and we can only have one default export.